openeogeotrellis/geopysparkcubemetadata.py

```diff
@@ -0,0 +1,153 @@
+import logging
+from typing import List, Union
+
+from openeo.metadata import CollectionMetadata, Dimension
+from openeogeotrellis.utils import reproject_cellsize
+
+_log = logging.getLogger(__name__)
+
+
+def clean_2D_tuple(tuple_to_clean):
+    if not tuple_to_clean:
```

Comment:
what is 2D about the returned value?

Reply:
Comment added: "Convert input an (x,y) tuple if possible."

Reply:
no I mean, (x,y) is not a 2D tuple, it is a 1D tuple with 2 items

Reply:
yeah "tuple2" or "pair" make more sense I think

Reply:
clean_number_pair